scene/__init__.py

```diff
@@ -34,6 +35,7 @@ def __init__(self):
         self.current_scene = None
         self._scene_cache = {}
         self._scene_time_stamp = 0
+        self.offset = repeat((0,0))
```

Comment:
does `offset` ever get changed back to this? seems like for consistency and clarity you'd want to use `self.offset = self.screen_shake(0, 0)`

Reply:
This was just setting up a default, but I like your suggestion better. I can refactor `screen_shake` out into its own utility file and then reference it both here and in `watermelon.py`